media/com_finder/js/index.es6.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+Joomla = window.Joomla || {};
+
+(() => {
+  'use strict';
+
+  document.addEventListener('DOMContentLoaded', () => {
+    Joomla.submitbutton = (pressbutton) => {
+      if (pressbutton === 'index.purge') {
+        // TODO replace with joomla-alert
+        if (window.confirm(Joomla.JText._('COM_FINDER_INDEX_CONFIRM_PURGE_PROMPT'))) {
+          Joomla.submitform(pressbutton);
+        } else {
+          return false;
+        }
+      } if (pressbutton === 'index.delete') {
+        // TODO replace with joomla-alert
+        if (window.confirm(Joomla.JText._('COM_FINDER_INDEX_CONFIRM_DELETE_PROMPT'))) {
+          Joomla.submitform(pressbutton);
+        } else {
+          return false;
+        }
+      }
+      Joomla.submitform(pressbutton);
+      return true;
```

Comment:
Same as in https://github.com/joomla-projects/joomla-es6/pull/90#pullrequestreview-102567078